src/main/java/frc/robot/subsystems/vision/PhotonSubsystem.java

```diff
@@ -125,24 +127,49 @@ public double getTurretYaw(int id) {
       List<PhotonTrackedTarget> targets = turretResult.getTargets();
       for (PhotonTrackedTarget target : targets) {
         if (target.getFiducialId() == id) {
-          return target.getYaw();
+          lastRecordedYawTime = turretResult.getTimestampSeconds();
+          lastRecordedYaw = target.getYaw();
+          return lastRecordedYaw;
         }
       }
     }
-    return Double.NaN;
+    if (lastRecordedYawTime != 0
+        && lastRecordedYawTime + Constants.LimelightConstants.APRILTAG_STALE_TIME
+            < Timer.getFPGATimestamp()) {
+      return lastRecordedYaw;
+    } else {
+      lastRecordedYawTime = 0;
+      lastRecordedYaw = 0;
+      return Double.NaN;
+    }
   }
 
-  public double getTurretPitch(int id) {
+  // FIXME: DISTANCE RETURN IS INCONSISTENT
+  // AND NOT ACCURATE
```

Comment:
I meant the LimelightSubsystem one, since we didn't update that one from getTurretPitch()